### ISSUE TITLE
Set up Node.js/Express.js

### ISSUE THREAD

**Issue by andrewzulaybar** [ubclaunchpad/foodies#3]
Task: get a basic "Hello, world!" level boilerplate running for Node.js and Express.js in TypeScript.

If you could integrate a testing library that would be great. I'm leaning towards [Jest](https://jestjs.io/) cause it's a pretty popular choice, but if you have some other ones that you'd prefer please bring them up here! I also suggested in #2 to have `yarn` set up instead of `npm` on the front-end so it would be nice to have that on the back-end as well.

Feel free to make suggestions on things we should add at the start, but I think this should be good for now. We can add other libraries later on as we see fit.

**Comment by ericrkuo** [comment]
Would this kind of be our folder structure in the future? Can't commit these empty directories, not until we add files into them.
https://git.wiki.kernel.org/index.php/GitFaq#Can_I_add_empty_directories.3F
![image](https://user-images.githubusercontent.com/49849754/96822977-305dfc80-13e0-11eb-983a-a97a432e02b3.png)

**Comment by ld357** [comment]
I've worked with Jest before and liked it! Since we're working with Typescript, I added [tsjest](https://github.com/kulshekhar/ts-jest) instead!
